github/GithubObject.py

```diff
@@ -324,7 +324,7 @@ def format_params(params: Dict[str, Any]) -> typing.Generator[str, None, None]:
     def _initAttributes(self) -> None:
         raise NotImplementedError("BUG: Not Implemented _initAttributes")
 
-    def _useAttributes(self, attributes: Dict[str, Any]) -> None:
+    def _useAttributes(self, attributes: Any) -> None:
         raise NotImplementedError("BUG: Not Implemented _useAttributes")
 
```

Comment:
This looks like a good idea but doesn't rename `_useAttributes` to `_useDictAttributes` may be misleading

Reply:
after second through, I don't think `_useTupleAttributes` is a good idea, this 
added additional complexity. using `_useAttributes` is easy to understand and easy to impl.

Reply:
But `_useAttributes(Any)` is pretty brutal. Any other suggestions to avoid `Any`? The implementing classes should be specific about their `attributes` parameter.

Can they define it as eihter `Dict[str, Any]` or `Tuple[Any, ...]`?

Reply:
I don't think it's a problem, only impl class know what type the value is.